bin/build-docs.sh

```diff
@@ -9,23 +9,19 @@ then
   exit
 fi
 
-rootDir=$PWD
-
-inputDir="docs/en"
+baseDir=$(dirname "$(dirname "$0")")
```

Comment:
Maybe the way to make this work in all cases would be to go up one directory until you find a `composer.json` 🤔 🤔 🤔